sql/hive/src/main/scala/org/apache/spark/sql/hive/client/HiveClientImpl.scala

```diff
@@ -365,8 +365,8 @@ private[hive] class HiveClientImpl(
     Option(client.getTable(dbName, tableName, false)).map { h =>
       // Note: Hive separates partition columns and the schema, but for us the
       // partition columns are part of the schema
-      val partCols = h.getPartCols.asScala.map(fromHiveColumn)
-      val schema = StructType(h.getCols.asScala.map(fromHiveColumn) ++ partCols)
+      val partCols = h.getPartCols.asScala.map(HiveClientImpl.fromHiveColumn)
```

Comment:
Add ```import HiveClientImpl._``` at the beginning of `class HiveClientImpl`. Then, you can avoid adding `HiveClientImpl` in the class `HiveClientImpl `